includes/Traits/Assets.php

```diff
@@ -44,15 +44,18 @@ trait Assets {
 	protected function get_asset_metadata( $handle ) {
 		$base_path = WEBSTORIES_PLUGIN_DIR_PATH . 'assets/js/';
 
+		// *.asset.php is generated by DependencyExtractionWebpackPlugin.
+		// *.chunks.php is generated by HtmlWebpackPlugin with a custom template.
 		$asset_file  = $base_path . $handle . '.asset.php';
 		$chunks_file = $base_path . $handle . '.chunks.php';
 		$asset       = is_readable( $asset_file ) ? require $asset_file : [];
 		$chunks      = is_readable( $chunks_file ) ? require $chunks_file : [];
 
 		$asset['dependencies'] = isset( $asset['dependencies'] ) ? $asset['dependencies'] : [];
-		$asset['version']      = isset( $asset['version'] ) ? $asset['version'] : WEBSTORIES_VERSION;
-		$asset['js']           = isset( $chunks['js'] ) ? $chunks['js'] : [];
-		$asset['css']          = isset( $chunks['css'] ) ? $chunks['css'] : [];
+		// A hash calculated based on the file content of the entry point bundle at <$handle>.js.
+		$asset['entry_version'] = isset( $asset['version'] ) ? $asset['version'] : WEBSTORIES_VERSION;
```

Comment:
We can use `$entry_version `later on, but here in the array it should stay just `version`